control/control.openSUSE.xml

```diff
@@ -297,6 +297,10 @@ textdomain="control"
                         <path>boot/grub2/arm64-efi</path>
                         <archs>aarch64</archs>
                     </subvolume>
+		    <subvolume>
+                        <path>boot/grub2/arm-efi</path>
+                        <archs>armv7l,armv7hl</archs>
```

Comment:
These values for `archs` should match any of the methods provided by the YaST `Arch` module. Currently those are: i386, sparc32, sparc64, sparc, ppc32, ppc64, ppc, alpha, s390_32, s390_64, s390, ia64, x86_64, arm, aarch64, board_compatible, board_mac, board_mac_new, board_mac_old, board_chrp, board_powernv, board_iseries, board_prep, board_pegasos, board_wintel.

Reply:
Fixed to use `arm`.